### ISSUE TITLE
Add .jars to a lib directory, split project into modules, and add build files

### ISSUE THREAD

**Issue by romanalexander** [MarkehMe/FactionsFramework#2]
Makes it easier for others to compile. Some of these old factions versions are hard to hunt down, and there's no proper maven for MassiveCore.

**Comment by MarkehMe** [comment]
Sure! I can do this. Doesn't simplify compiling though (ugh), will have to create a guide to compiling.

**Comment by romanalexander** [comment]
I think we can take a chapter out of sudzzy's playbook with FBasics. Separate version out and apply dependencies per module.
https://github.com/Sudzzy/FBasics/blob/master/pom.xml

**Comment by MarkehMe** [comment]
Sounds like a good idea actually! I will edit my first comment with a todo list for this ticket.

**Comment by MarkehMe** [comment]
So I've added Gradle support and moved it into modules. Need to fix some small build issues i've created but there are some small scripts i've created to help with these.

**Comment by MarkehMe** [comment]
Modules are building fine under Gradle. I will leave it at Gradle support and work on other issues from now so I can push out 1.1.0.

**Comment by MarkehMe** [comment]
🎉  **FactionsFramework v1.1.0 has been released and addresses this issue** 🎉 

You can grab it from:
* [GitHub](https://github.com/MarkehMe/FactionsFramework/releases/tag/1.1.0)
* [Spigot Resources](https://www.spigotmc.org/resources/factions-framework.22278/update?update=89455)
* [BukkitDev](http://dev.bukkit.org/bukkit-plugins/factionsframework/files/2-factions-framework-v1-1-0/)

```
v1.1.0 Changelog:

* Factions#getNone() has been deprecated and will be unavailable in 6 months
* Factions#getNone(World) added
* Factions#getSafezone(World) added
* Factions#getWarzone(World) added
* Stability fixes for 2.8.6, 2.8.2, 2.7, 2.6
* Supports FactionsOne (1.8)
* Uses a gradle module layout (less reflection now too!)
* Internal changes and performance enhancements
```
